packages/target-tauri/src-tauri/tauri.conf.json5

```diff
@@ -26,10 +26,10 @@
         "style-src": "'self' 'unsafe-inline'",
         "font-src": "'self'",
         "script-src": "'self' 'wasm-unsafe-eval'",
-        "worker-src": "blob: ",
-        "child-src": "blob: ",
-        "img-src": "'self' data: blob: dcblob: webxdc-icon: ",
-        "media-src": "'self' dcblob:"
+        "worker-src": "blob: http://blob.localhost",
+        "child-src": "blob: http://blob.localhost",
+        "img-src": "'self' data: blob: http://blob.localhost dcblob: http://dcblob.localhost webxdc-icon: http://webxdc-icon.localhost",
+        "media-src": "'self' dcblob: http://dcblob.localhost"
```

Comment:
Is there no other way? This practically enables access to all localhost HTTP servers. Try `python3 -m http.server 8080`, then go to http://blob.localhost:8080 : you'll get to the server.

Reply:
maybe there is a way to only enable it for windows & android

Reply:
I'll add this to the my todo list, we will fix it later